### ISSUE TITLE
Falling back to default lang when iterating on an array

### ISSUE THREAD

**Issue by HugoCrd** [kurtsson/jekyll-multiple-languages-plugin#98]
Hi!

First, thanks a lot for this very useful gem.

I'm looking for a way to iterate on an array and falling back to `site.default_lang` if `site.lang` does not contain the translation.
The `for o in site.translations[site.lang].somethings` solution does work, but not if I want to fall back to default lang.

For now I came with the following solution which I don't find very elegant:

```
{% if site.translations[site.lang].blocks %}
  {% assign blocks = site.translations[site.lang].blocks %}
{% else %}
  {% assign blocks = site.translations[site.default_lang].blocks %}
{% endif %}

{% for block in blocks %}
   {{block}}
{% endfor %}
```

Is there a simpler way to achieve this that I did not think about?
If not, is this something you would be interesting in supporting?

Thanks again,
Hugo

**Comment by HanniAbu** [comment]
From what I've seen, the current version uses the default language as a fallback. Found this issue while trying to figure out how to *not* have that behavior.

**Comment by kurtsson** [comment]
Should be fixed by #130 right?
